test/unit/ParaTest/Console/Commands/ParaTestCommandTest.php

```diff
@@ -47,7 +47,9 @@ public function testConfiguredDefinitionWithPHPUnitTester()
             new InputOption('coverage-clover', null, InputOption::VALUE_REQUIRED, 'Generate code coverage report in Clover XML format.'),
             new InputOption('coverage-html', null, InputOption::VALUE_REQUIRED, 'Generate code coverage report in HTML format.'),
             new InputOption('coverage-php', null, InputOption::VALUE_REQUIRED, 'Serialize PHP_CodeCoverage object to file.'),
-            new InputOption('testsuite', null, InputOption::VALUE_OPTIONAL, 'Filter which testsuite to run')
+            new InputOption('testsuite', null, InputOption::VALUE_OPTIONAL, 'Filter which testsuite to run'),
+            new InputOption('max-batch-size', 'm', InputOption::VALUE_REQUIRED, 'Max batch size (only for functional mode).', 50),
```

Comment:
max-batch-size=1 as a default is preferrable for BC, isn't it?
Especially when the Readme contains information about performance optimization and max-batch-size.